src/cpu/aarch64/jit_uni_dw_conv_kernel_f32.hpp

```diff
@@ -40,7 +41,7 @@ struct jit_uni_dw_conv_fwd_kernel_f32_t : public jit_generator {
     DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_uni_dw_conv_fwd_kernel_f32_t)
 
     jit_uni_dw_conv_fwd_kernel_f32_t(jit_conv_conf_t ajcp)
-        : jcp(ajcp), eltwise_injector_(nullptr) {
+        : jcp(std::move(ajcp)), eltwise_injector_(nullptr) {
```

Comment:
Which check is this change from? I assume `performance-unnecessary-value-param`?